internal/mock/mock_eventclient.go

```diff
@@ -9,65 +9,67 @@ package mock
 import (
 	"context"
 
-	contract "github.com/edgexfoundry/go-mod-core-contracts/models"
+	"github.com/edgexfoundry/go-mod-core-contracts/models"
```

Comment:
we are using "contract" as the alias name for the contract model import in whole repository

Reply:
I'll readd the alias for consistency.